src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -62,40 +55,11 @@ public class LogsConfiguration : ConfigurationCategory
         [ConfigurationEntry("Logs.Microsoft.Disable", ConfigurationEntryScope.ServerWideOnly)]
         public bool DisableMicrosoftLogs { get; set; }
```

Comment:
Shouldn't this be set to 'false' with another configuration option of log level set to None?

Reply:
The disable here does nothing except to not call the `ApllyConfiguration` and ensure all future created logger will be set to None.
* In my observation all loggers are created before but just to be sure.

If no configuration is applied the default is `None`.
This is also the same for empty or a not exit config file.